components/Buttons/examples/ButtonsDynamicTypeViewController.swift

```diff
@@ -86,6 +79,13 @@ class ButtonsDynamicTypeViewController: UIViewController {
               options: .alignAllCenterX, metrics: nil, views: views))
   }
 
+  override func viewDidLayoutSubviews() {
+    super.viewDidLayoutSubviews()
+
+    let buttonFont = containerScheme.typographyScheme.button.mdc_scaledFont(for: view)
+    flatButtonDynamic.setTitleFont(buttonFont, for: .normal)
```

Comment:
Sanity checking here: I'd expect us to encourage setting `adjustsFontForContentSizeCategory` to YES after initializing the button rather than dynamically re-applying the font here. Can we do that instead here?

Reply:
+1.
should we remove the mdc_adjustsFontForContentSizeCategory calls too?

Reply:
I think the expectation is that in `viewDidLoad` the initial font is set using the `mdc_scaledFont(for:view)` call and that `mdc_adjustsFontForContentSizeCategory` is also set. After that, there should be no need to do anything in `viewDidLayoutSubviews` for Typography.

Reply:
We should not be setting `adjustsFontForContentSize` here since it has no effect on `MDCFontScaler` fonts.

Reply:
ok I think that I understand now.
echoing back to you: The 3 version of this is not ready yet so we have to use v2: mdc_adjustsFontForContentSizeCategory calls.